tests_metricflow/snapshots/test_metric_filter_rendering.py/SqlQueryPlan/BigQuery/test_metric_filtered_by_itself__plan0_optimized.sql

```diff
@@ -0,0 +1,39 @@
+-- Constrain Output with WHERE
+-- Pass Only Elements: ['bookers',]
+-- Aggregate Measures
+-- Compute Metrics via Expressions
+SELECT
+  COUNT(DISTINCT bookers) AS bookers
+FROM (
+  -- Join Standard Outputs
+  -- Pass Only Elements: ['bookers', 'guest__bookers']
+  SELECT
+    subq_26.bookers AS guest__bookers
+    , subq_20.bookers AS bookers
+  FROM (
+    -- Read Elements From Semantic Model 'bookings_source'
+    -- Metric Time Dimension 'ds'
+    -- Pass Only Elements: ['bookers', 'guest']
+    SELECT
+      guest_id AS guest
+      , guest_id AS bookers
+    FROM ***************************.fct_bookings bookings_source_src_28000
+  ) subq_20
+  LEFT OUTER JOIN (
+    -- Read Elements From Semantic Model 'bookings_source'
+    -- Metric Time Dimension 'ds'
+    -- Pass Only Elements: ['bookers', 'guest']
+    -- Aggregate Measures
+    -- Compute Metrics via Expressions
+    -- Pass Only Elements: ['guest', 'bookers']
+    SELECT
+      guest_id AS guest
+      , COUNT(DISTINCT guest_id) AS bookers
+    FROM ***************************.fct_bookings bookings_source_src_28000
+    GROUP BY
+      guest
+  ) subq_26
+  ON
+    subq_20.guest = subq_26.guest
+) subq_28
+WHERE guest__bookers > 1.00
```

Comment:
LOL. This is maybe not the best example query since the value of `bookers` will always be 1 here, but it's fine.